rdt/transformers/base.py

```diff
@@ -255,16 +255,25 @@ def get_output_columns(self):
         return list(self._get_output_to_property('sdtype'))
 
     def _store_columns(self, columns, data):
-        if isinstance(columns, tuple) and columns not in data:
-            columns = list(columns)
-        elif not isinstance(columns, list):
-            columns = [columns]
+        """Store the column names to be transformed.
 
-        missing = set(columns) - set(data.columns)
+        Args:
+            columns (str, list, tuple):
+                Column name(s) to be transformed.
+            data (pandas.DataFrame):
+                The entire table.
+        """
+        column_names = columns
```

Comment:
what's the point of this assignment? Why not just change the name of the variable

Reply:
Yes first I was not sure if we really wanted to modify the column parameter, but I removed my change, it was not necessary